File: src/main/java/com/sotasan/decompiler/menus/help/Help.java
package com.sotasan.decompiler.menus.help;

import com.formdev.flatlaf.extras.components.FlatMenu;
import com.sotasan.decompiler.services.LanguageService;
import java.awt.*;
import java.awt.event.KeyEvent;

public class Help extends FlatMenu {

    public Help() {
        setMnemonic(KeyEvent.VK_H);
        setText(LanguageService.getTranslation("help"));

        add(new HelpAbout());

        if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.APP_ABOUT))
            setVisible(false);
    }

}

File: src/main/kotlin/moe/sota/decompiler/menus/help/Help.kt
package moe.sota.decompiler.menus.help

import com.formdev.flatlaf.extras.components.FlatMenu
import moe.sota.decompiler.services.LanguageService
import java.awt.Desktop
import java.awt.event.KeyEvent

class Help(
    about: HelpAbout
) : FlatMenu() {

    init {
        isVisible = !(Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.APP_ABOUT))
        mnemonic = KeyEvent.VK_H
        text = LanguageService.getTranslation("help")

        add(about)
    }

}
